config/samples/core_v1beta1_openstackcontrolplane_collapsed_cell.yaml

```diff
@@ -129,16 +129,29 @@ spec:
       metricStorage:
         enabled: false
         monitoringStack:
-          type: prometheus
+          alertingEnabled: true
+          scrapeInterval: 30s
+          storage:
+            strategy: persistent
+            retention: 24h
+            persistent:
+              pvcStorageRequest: 20G
       autoscaling:
         enabled: false
-        prometheus:
-          deployPrometheus: false
         aodh:
+          passwordSelectors:
+          databaseUser: aodh
           databaseInstance: openstack
           memcachedInstance: memcached
           secret: osp-secret
         heatInstance: heat
       ceilometer:
         enabled: true
         secret: osp-secret
+      logging:
+        enabled: false
+        network: internalapi
+        ipaddr: 172.17.0.80
+        port: 10514
+        cloNamespace: openshift-logging
+
```

Comment:
Linting probably doesn't want this blank line, otherwise looks good

Reply:
Fixing it right now